packages/primitives/src/index.js

```diff
@@ -8,6 +8,12 @@ export const Note = 'NOTE';
 export const Path = 'PATH';
 export const Rect = 'RECT';
 export const Line = 'LINE';
+export const Form = 'FORM';
+export const FormField = 'FORM_FIELD';
+export const TextInput = 'TEXT_INPUT';
+export const FormPushButton = 'FORM_PUSH_BUTTON';
+export const Picker = 'PICKER';
+export const FormList = 'FORM_LIST';
```

Comment:
What do you think of [keeping the initial names of these components](https://github.com/diegomura/react-pdf/pull/2013/files#r1014982991) to keep them aligned with the form annotation methods of `PDFKit`?

```
formText( name, x, y, width, height, options)
formPushButton( name, x, y, width, height, name, options)
formCombo( name, x, y, width, height, options)
formList( name, x, y, width, height, options)
```
([src](https://pdfkit.org/docs/forms.html#form_annotation_methods))